brave/brave-core
# Fixes an Add Widget button on Android 8 devices

Resolves https://github.com/brave/brave-browser/issues/27825

## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

1. Make a fresh install of Brave on Android 8
2. Observe that the Add Widget button from the issue looks good.
